bin/setup

```diff
@@ -17,6 +17,9 @@ FileUtils.chdir APP_ROOT do
   puts "== Installing dependencies =="
   system("bundle check") || system!("bundle install")
 
+  # Install JavaScript dependencies
+  system("yarn install --check-files")
+
```

Comment:
We use `bundle exec bin/yarn install` in the install notes, Docker file, and vagrant provisioning, and so I would expect this to be consistent with those.